src/reanimated2/scrollValueSetter.ts

```diff
@@ -0,0 +1,64 @@
+import { AnimationObject } from './commonTypes';
+import { scrollTo } from './NativeMethods';
+export { stopMapper } from './mappers';
+
+export function scrollValueSetter(sv: any, value: any, animatedRef: any): void {
```

Comment:
This seems to be copied to some extent from the default value setter. I wonder if this can just be done using mutable's listeners?

Reply:
Initially I did it with listener, but it cause a cycle or unnecessary calls.
We want to call `scrollTo` method when:
- public constructor is called. 
- there is a next iteration of the animation

With a listener approach we also call `scrollTo` when:
- the value is changed because of the previous `scrollTo`.
- the user drags `ScrollView`

The alternative approach can be adding 3rd setter to differentiate event assignment, but I think that my solution is more clear.

Maybe I don't see the better solution, if you have any suggestions, I'm happy to hear it.